includes/degree-functions.php

```diff
@@ -250,17 +238,79 @@ function get_degree_application_deadline_modern_layout() {
 	 $application_deadline_second = get_field( 'application_deadline_second' );
 
 	 if( empty( $application_deadline_first ) && empty( $application_deadline_second ) ) return '';
- 
+
 	 ob_start();
 	 ?>
-	 <div class="bg-default">
-		<div class="container py-lg-3">
-			<div class="row my-lg-3">
-				<div class="col text-center text-uppercase">
-					<?php echo $application_deadline_first; ?><br>
+	<div class="ucf-section" aria-label="Application Deadline">
+		<div class=" d-block d-lg-none">
+			<div class="bg-primary text-center pt-5">
+				<h2 class="h4 text-uppercase font-condensed my-0">Application Deadline</h2>
+		<?php if( $application_deadline_first ) : ?>
+			</div>
+			<div class="text-uppercase text-center bg-primary py-4">
+				<?php echo $application_deadline_first; ?>
+			</div>
+		<?php endif; ?>
+		<?php if( $application_deadline_second ) : ?>
+			<div class="bg-primary py-1">
+				<hr class="hr-white hr-2 my-0 application-deadline-hr">
+			</div>
+			<div class="text-uppercase text-center bg-primary py-4 bg-arrow-down">
+				<div class="pb-5">
 					<?php echo $application_deadline_second; ?>
 				</div>
 			</div>
+		<?php endif; ?>
+			<div class="modern-degree-dark-grey text-white py-4 text-center">
+				<h2 class="h5 text-uppercase font-condensed mt-3">Ready to<br>get started?</h2>
+			</div>
+			<div class="col pt-1 pb-5 text-uppercase modern-degree-dark-grey text-center">
+				<a id="CTA" class="btn btn-lg btn-primary rounded" rel="noopener noreferrer" data-toggle="modal" data-target="#formModal">Apply Now</a>
+			</div>
+		</div>
+	</div>
+
+	<div class="container-fluid px-0 d-none d-lg-block">
```

Comment:
The desktop version of this content should be included in the same `section` as the mobile content